Tests/SVT-HEVC_FunctionalTests.py

```diff
@@ -1090,40 +1398,47 @@ def run_speed_test(self, seq_dict):
             print("", file=open('speed_script.bat', 'w'))
         else:
             print("", file=open('speed_script.bat', 'w'))
-        for seq in seq_dict:
-            enc_params.update(self.get_stream_info(seq['name']))
-            if 'qp' in seq and 'tbr' in seq:
-                print("Please only have \"qp\" or \"tbr\" for each sequence")
-                continue
-            elif not 'qp' in seq and not 'tbr' in seq:
-                print("Please have \"qp\" or \"tbr\" for each sequence")
-                continue
-            elif 'qp' in seq:
-                VBR = 0
-                iter_list = seq['qp']
-            elif 'tbr' in seq:
-                VBR = 1
-                iter_list = seq['tbr']
-            if VBR == 1:
-                continue
-            for enc_mode in SPEED_ENC_MODES:
-                num_channels = self.get_num_channels(enc_mode, enc_params)
-                num_frames = self.get_num_frames(enc_mode)
-                if VBR == 0:
-                    enc_params.update({'enc_mode': enc_mode, 'qp': iter_list, 'rc': 0, 'frame_to_be_encoded': num_frames})
-                    cmd = self.get_enc_cmd(enc_params, seq['name'], 'Speed_Test_M' + str(enc_mode) + '_' + seq['name'] + '_Q' + str(iter_list), num_channels)
-                else:
-                    enc_params.update({'enc_mode': enc_mode, 'tbr': iter_list, 'rc': 1, 'frame_to_be_encoded': num_frames})
-                    cmd = self.get_enc_cmd(enc_params, seq['name'], 'Speed_Test_M' + str(enc_mode) + '_' + seq['name'] + '_TBR' + str(iter_list), num_channels)
-                print (cmd)
-                if platform == WINDOWS_PLATFORM_STR:
-                    print(cmd, file=open('speed_script.bat', 'a'))
-                else:
-                    print(cmd, file=open('speed_script.sh', 'a'))
+        for OQ in SQ_OQ_COMBINATION:
```

Comment:
SQ_OQ_COMBINATION is not defined anywhere.
Btw, there is no -tune mode anymore. SQ, OQ, VMAF is removed during 1.4 release.

Reply:
Let's verify with Jing

Reply:
please remove OQ/SQ tune mode, and keep original github "def run_speed_test(self, seq_dict) " function.